specification/search/resource-manager/Microsoft.Search/stable/2015-08-19/search.json

```diff
@@ -1000,6 +1004,35 @@
           "description": "The URL to get the next set of operation list results, if any."
         }
       }
+    },
+    "Identity": {
+      "properties": {
+        "principalId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The principal ID of resource identity."
+        },
+        "tenantId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The tenant ID of resource."
+        },
+        "type": {
+          "type": "string",
+          "description": "The identity type.",
```

Comment:
Is there more detail we can add here so the generated documentation will be more self-explanatory?